meta-resin-common/recipes-containers/docker/docker_git.bb

```diff
@@ -123,8 +115,15 @@ SYSTEMD_SERVICE_${PN} = "docker.service var-lib-docker.mount"
 
 do_install() {
   mkdir -p ${D}/${bindir}
-  install -m 0755 ${S}/bundles/${DOCKER_VERSION}/dynbinary/docker-${DOCKER_VERSION} \
-    ${D}/${bindir}/docker
+  install -m 0755 ${S}/bundles/${DOCKER_VERSION}/binary-rce/rce ${D}/${bindir}/rce
+
+  ln -sf ${bindir}/rce ${D}/${bindir}/docker
```

Comment:
The convention for such links is to not use an absolute path so they should be changed to 
```bash
ln -s rce ${D}/${bindir}/docker
```